frontend/package.json

```diff
@@ -6,24 +6,24 @@
     "types": "dist/types/index.d.ts",
     "private": true,
     "dependencies": {
-        "@cloudscape-design/components-themeable": "^3.0.546",
-        "@cloudscape-design/components": "^3.0.341",
-        "@cloudscape-design/global-styles": "^1.0.10",
-        "@cloudscape-design/design-tokens": "^3.0.30",
+        "@cloudscape-design/components": "^3.0.580",
+        "@cloudscape-design/components-themeable": "^3.0.595",
+        "@cloudscape-design/design-tokens": "^3.0.34",
+        "@cloudscape-design/global-styles": "^1.0.25",
         "@reduxjs/toolkit": "^1.9.5",
-        "axios": "^0.27.2",
+        "axios": "^0.28.0",
         "git-repo-info": "^2.1.1",
-        "jest-mock-axios": "^4.7.0-beta",
+        "jest-mock-axios": "^4.7.3",
         "lodash": "4.17.21",
-        "quill": "^1.3.5",
+        "quill": "^2.0.0-rc.3",
```

Comment:
Did you test the WYSIWYG editor for the create Groundtruth job page still works after this? Seems like 1.3.5 -> 2.0.0 is a big jump.

Reply:
![image](https://github.com/awslabs/mlspace/assets/28429388/42f929e3-6437-4225-a1fe-26479e61a555)

Turns out this does break it. I need to figure out how to fix this

Reply:
As discussed in internal channel, I will revert this change and create a backlog item for updating quill to v2